collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryViewModel.java

```diff
@@ -71,14 +71,14 @@ public void addRepeat(boolean fromPrompt) {
     public void cancelRepeatPrompt() {
         analytics.logEvent(ADD_REPEAT, "InlineDecline");
 
-        FormController formController = getFormController();
+        FormController formController = this.formController;
```

Comment:
Not necessary?

Reply:
haha yeah. I'd imagine AS probably flagged that to me as well!